tests/acceptance/ExportSubscribersCest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+use MailPoet\Test\DataFactories\Segment;
+
+require_once __DIR__ . '/../DataFactories/Segment.php';
+require_once __DIR__ . '/../_data/MailPoetExportList.csv';
+
+class ExportSubscribersCest {
+  function __construct() {
+    $this->search_field_element = 'input.select2-search__field';
+  }
+  function exportSubscribers(\AcceptanceTester $I) {
+    $segment_name = 'Hobbyists';
+    $segment = new Segment();
+    $segment->withName($segment_name)->create();
+    $I->wantTo('Export a list of subscribers');
+    $I->login();
+    $I->amOnMailPoetPage('Subscribers');
+    //import unique user list to export, so we have exact data regardless of other tests
+    $I->click(['xpath'=>'//*[@id="subscribers_container"]/div/h1/a[2]']);
+    $I->waitForText('Back to Subscribers', 10);
+    //select upload file as import method, import CSV
+    $I->click(['css'=>'#select_method > label:nth-of-type(2)']);
+    $I->attachFile(['css'=>'#file_local'], 'MailPoetExportList.csv');
```

Comment:
I see that you're attempting to re-import some of the subscribers so that you have something to export, I understand the reasoning here.
However, in a way that also tests subscriber import. We should have a separate test for import, this test should focus only on the export functionality.

Instead of handling it via the user interface, I would suggest to use DataFactories to create several specific users in a new list, and try exporting that.
That way the test itself should be a bit faster as well, and it will test only what is intended - subscriber export.